test/views/HubViewSpec.scala

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import base.SpecBase
+import models.{CertificationDetails, CompanyDetails, NotificationDetails}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Element
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.i18n.{Messages, MessagesApi}
+import play.api.mvc.Request
+import play.api.test.FakeRequest
+import views.html.HubView
+
+import java.time.LocalDate
+
+class HubViewSpec extends SpecBase with GuiceOneAppPerSuite {
+  val SUT: HubView = app.injector.instanceOf[HubView]
+
+  given request: Request[_] = FakeRequest()
+
+  given Messages = app.injector.instanceOf[MessagesApi].preferred(request)
+
+  private val testDate = LocalDate.of(2025, 7, 30)
+
+  val companyDetails = CompanyDetails(
+    companyName = "Fake Company Ltd",
+    referenceId = "fakexxx1234",
+    accountingPeriodStartDate = testDate,
+    accountingPeriodEndDate = testDate
+  )
+
+  val notificationDetails = NotificationDetails(
+    dueDate = testDate
+  )
+
+  val certificationDetails = CertificationDetails(
+    dueDate = testDate
+  )
+  val doc         = Jsoup.parse(SUT(companyDetails, notificationDetails, certificationDetails).toString)
+  val mainContent = doc.getElementById("main-content")
+
+  "HubView" must {
+
+    "must generate a view with the correct heading and title" in {
+      val h1 = mainContent.getElementsByTag("h1")
+      h1.size() mustBe 1
+      h1.get(0).text() mustBe "Senior Accounting Officer notification and certificate account"
+      doc.title mustBe "Senior Accounting Officer notification and certificate account - Senior Accounting Officer notification and certificate account - site.govuk"
+    }
+
+    "must have correct correct number of sections" in {
+      val sections = mainContent.getElementsByAttributeValueContaining("id", "section-")
+      sections.size() mustBe 5
+    }
+
+    "must have correct content for company details section" in {
+      val rows = mainContent.getElementById("section-company-details").select("div.govuk-summary-list__row")
+      rows.size() mustBe 3
+      validateRow(rows.get(0), keyText = "Company name", actionText = "Fake Company Ltd", None)
```

Comment:
Can we update all of the call site of validateRow with all of the parameter names and remove the Nones (since this is now defaulted to None)
```suggestion
      validateRow(row = rows.get(0), keyText = "Company name", actionText = "Fake Company Ltd")
```